LEGO1/mxtransitionmanager.h

```diff
@@ -2,19 +2,58 @@
 #define MXTRANSITIONMANAGER_H
 
 #include "mxcore.h"
+#include "mxvideopresenter.h"
+#include "legovideomanager.h"
+#include "legoomni.h"
 
-class MxVideoPresenter;
 
-// 0x100d7ea0
+// VTABLE 0x100d7ea0
```

Comment:
Could you add the current size of the class and a `DECOMP_SIZE_ASSERT`? Thank you!